### ISSUE TITLE
No batch option to run over a group or directory of models

### ISSUE THREAD

**Issue by jebarpg** [AUTOMATIC1111/stable-diffusion-webui-tensorrt#30]
It would be more effective to be able to batch process models with a set configuration for all models that will be converted. This way you can set it and forget it over night or days until the batch of models has been processed. 

Sorry maybe not an issue, but didn't know where to mention this.

**Comment by wizz13150** [comment]
Yep, try this : https://github.com/AUTOMATIC1111/stable-diffusion-webui-tensorrt/pull/36
